docs/source/teams/installation.rst

```diff
@@ -72,6 +78,51 @@ That's it! Any operations you perform will be stored in a centralized location
 and will be available to all users with access to the same datasets in the
 Teams App or their Python workflows.
 
+.. _teams-installation-poetry:
+
+Installation with Poetry
+________________________
+If you  are using `poetry <https://python-poetry.org/>`_ to install your
+dependencies rather than ``pip``, you will need to follow instructions in
+`the docs for installing from a private repository. <https://python-poetry.org/docs/repositories/#installing-from-private-package-sources>`_
+The two key points are specifying the additional private source and declaring
+that the ``fiftyone`` module should be found there and not the default PyPI
+location.
+
+In poetry v1.5, it is recommended to use an
+`explicit package source. <https://python-poetry.org/docs/repositories/#explicit-package-sources>`_
+Prior to v1.5, you should use a
+`secondary package source. <https://python-poetry.org/docs/repositories/#secondary-package-sources-deprecated>`_
+
+For example,
+
+.. code-block:: shell
+
+    poetry source add --priority=explicit fiftyone-teams https://pypi.fiftyone.ai/simple
+
+    export POETRY_HTTP_BASIC_FIFTYONE_TEAMS_USERNAME="${TOKEN}"
+    export POETRY_HTTP_BASIC_FIFTYONE_TEAMS_PASSWORD=""
+
+    poetry add --source fiftyone-teams fiftyone
+
+If you have trouble configuring the credentials, see
+`more in the docs here. <https://python-poetry.org/docs/repositories/#configuring-credentials>`_
+
+You should then see snippets in the ``pyproject.toml`` file like the following:
+
+.. code-block:: toml
+
+    [[tool.poetry.source]]
+    name = "fiftyone-teams"
+    url = "https://pypi.fiftyone.ai/simple/"
+    default = false
+    priority = explicit
```

Comment:
This snippet actually won't work with 1.5 any longer:
```
[[tool.poetry.source]]
name = "fiftyone-teams"
url = "https://pypi.fiftyone.ai/simple/"
priority = "explicit"
```

is the new swagger... (and will throw an error if you use the old syntax) 

might have to differentiate for different versions?  Or just not show them the innards?